test/functional/test_transformer_implementation.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+import numpy as np
+
+from sagemaker_containers import content_types, env, serializers, status_codes, transformers, worker
+import test
+from test import miniml
+
+
+class MiniMlTransformer(transformers.BaseTransformer):
+    def predict_fn(self, model, data):
+        return model.predict(data)
+
+    def model_fn(self, model_dir):
+        return miniml.Model.load(os.path.join(model_dir, 'minimlmodel'))
+
+
+def test_prediction_without_user_module():
```

Comment:
is there a test _with_ user module?

Reply:
There is, in the unit/test_transfomers. I forgot to rename this test.